inference-engine/thirdparty/movidius/mvnc/src/mvnc_api.c

```diff
@@ -424,11 +424,7 @@ static ncStatus_t getDeviceFwProtocolPrefix(const deviceDesc_t deviceDesc,
 
 static char* getDevicePlatform(deviceDesc_t deviceDesc, int useUniversalFirmware) {
```

Comment:
It looks like `useUniversalFirmware` is now an unused argument

Reply:
It was not a good solution to return ma2x8x always. We still can work with ma248x. I reverted some changes and removed special conditional for pcie